charts/github-exporter/templates/secret.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
   name: {{ include "github-exporter.fullname" . }}
   namespace: {{ .Release.Namespace }}
 data:
-  GITHUB_TOKEN: {{ .Values.secret.githubToken | b64enc }}
+  GITHUB_API_TOKEN: {{ .Values.secret.githubToken | b64enc }}
```

Comment:
we can change to be all `GITHUB_TOKEN` which is more standard, i dont remember why i set to GITHUB_API_TOKEN

is it possible you do that change? otherwise i can do it :)

Reply:
I changed it, but I'm wondering if it's OK to introduce a breaking change with the token naming. What do you think?

Reply:
Ah ok, i can do in a follow up when we have other changes thanks